rock_paper_scissor.py

```diff
@@ -0,0 +1,106 @@
+import random
+
+
+class RpsGame:
+    def __init__(self, max_attempts):
+        self._current_attempt = 0
+        self._max_attempts = max_attempts
+        self._comp_choice = None
+        self.user_score = 0
+        self.comp_score = 0
+
+    def increment_user_score(self):
+        self.user_score += 1
+
+    def increment_comp_score(self):
+        self.comp_score += 1
+
+    def is_attempts_left(self) -> bool:
+        return self._current_attempt < self._max_attempts
+
+    def make_choice(self):
+        self._current_attempt += 1
+        self._comp_choice = random.choice(["1", "2", "3"])
+
+    def get_comp_choice(self) -> str:
+        return self._comp_choice
+
+    def get_result(self, user_input: str) -> str:
+        results_mapping = {
+            "1": {
+                "1": "DRAW",
+                "2": "LOSE",
+                "3": "WON",
+            },
+            "2": {
+                "1": "WON",
+                "2": ["DRAW"],
```

Comment:
I think this is unintentional here. Why are we returning a list for this specific case instead of string like all other cases.